manifests/global_defs.pp

```diff
@@ -131,6 +133,9 @@
   Optional[Stdlib::Absolutepath] $vrrp_notify_fifo   = undef,
   Optional[Stdlib::Absolutepath] $vrrp_notify_fifo_script = undef,
   $snmp_socket                                       = 'unix:/var/agentx/master',
+  Optional[
```

Comment:
should we add the `Optional` to the type alias? that makes the whole line shorter and can stick to a single line instead of three.